docs/safety.md

```diff
@@ -0,0 +1,150 @@
+- There is a larger fire extinguisher right outside 701
+
+*If something gets in the eye:*
+
+- There is an eye wash in each 701 sink
+- Do not rub eye
+- Call for help
+
+*If bleeding:*
+
+- Call for help
+- Apply pressure to the wound
+- First-aid kit is near the exit of 701
+- Wash hands
+- Use disinfectant
+- Use gauze/band-aids
+
+*If bleeding doesn't stop:*
+
+- Keep applying pressure
+- Go to urgent care
+
+*If someone loses consciousness:*
+
+- Call 911, **NO EXCEPTIONS**
+
+## Using Equipment
+
+You **must** wear goggles (and electric gloves if handling high
+voltage). This includes, but is not limited to: mechanical maintenance,
+using hand/power tools, cutting/stripping wire, soldering, etc...
+
+*When jacking a car:*
+
+- Wait until clear before jacking or releasing jack
+- Announce "ALL CLEAR, JACKING (FRONT OR BACK)"
+- Announce "ALL CLEAR, LOWERING" when releasing the jack
+- NO ONE allowed under car when it is on jacks (no exceptions)
+
+*When going under a car:*
+
+- Put up caution tape or assign someone else to watch the car so people
+  coming nearby know to stay clear of the car
+- Do not go under the car until you are sure it is safe to do so
+- Announce "GOING UNDER CAR" to make sure everyone near the car knows
+  not to sit inside or move the car
+
+*When operating experimental components:*
+
+- Confirm the component works on a lab bench before testing on a project
+- Ensure there is a safe way to disengage the component
+
+### When car is in 701:
+
+- Ensure car will not drive into wall (put car on dollies or use chock
+  blocks at all times)
+- Have the car face at the wall where it won’t hit the hallway or anyone
+  else in case it drives forward by itself
+- Avoid keeping the car on it’s wheels unless needed for testing (Use
+  jack stands as needed)
+
+*When operating power tools:*
+
+- Make sure all is clear before using the tool
+- Announce "ALL CLEAR, OPERATING (TOOL)" so that all know to be clear
+- Ensure items are properly clamped before drilling so they don't go
+  flying or break
+- Use step bits when possible for drilling into plastic
+
+*When wiring a battery:*
+
+- Make sure all is clear before removing insulation to prevent shorts
+- For anything loose use electric tape to prevent any accidental shorts
+- Use battery boots (where possible)
+- If a battery shorts, **stop** what you're doing, turn off all systems
+  to prevent further damage, and identify the cause for the short
+
+*When soldering:*
+
+- Ensure you have appropriate ventilation
+- Ensure soldering iron is not left on when finished
+
+*When using a heat gun:*
+
+- Only point at items you want to heat
+- Keep clear of people and other objects when the heat gun is on
+- Ensure the heat gun is turned off when finished
+
+## Test Day Procedures
+
+*In general:*
+
+- If not directly testing, **stay clear**
+- Do not distract people testing as it may cause a lapse in judgement
+- Save non-urgent questions for when people are not actively testing
+
```

Comment:
When arming vehicle, announce "ARMING VEHICLE"
Same when disarming vehicle.
When test is starting, announce "STARTING TEST"